tests/reftests/opamroot-versions.test

```diff
@@ -744,13 +744,12 @@ RSTATE                          LOAD-REPOSITORY-STATE @ ${BASEDIR}/OPAM
 RSTATE                          Cache found
 STATE                           LOAD-SWITCH-STATE @ sw-comp
 STATE                           Switch state loaded in 0.000s
-### opam-cat $OPAMROOT/config | '"${OPAMROOTVERSION}"' -> "current" | grep -v sys-pkg-manager-cmd | grep -v global-variables
+### opam-cat $OPAMROOT/config | '"${OPAMROOTVERSION}"' -> "current" | grep -v sys-pkg-manager-cmd | grep -v global-variables | grep -v eval-variables:
```

Comment:
I'm not sure about this `eval-variables` removal in the whole test, we may want to test that the `eval-variables` upgrade is well done, as it is the purpose of this test.

Reply:
Checked, it can be removed. I've added a specifically for those eval variables.